heareval/embeddings/runner.py

```diff
@@ -36,12 +54,16 @@ def runner(module: str, model: str = None, tasks_dir: str = "tasks") -> None:
         )
 
     # Load the embedding model
-    embedding = Embedding(module, model)
+    embedding = Embedding(module, model, str(version))
```

Comment:
Is this documented in the API anywhere? I mean maybe this makes sense but where is the API documentation? Otherwise I'm not sure we can just slip this change in without announcing it or whatever